### PR TITLE
[0.1.0] Landing and Categories

### PR DESCRIPTION
**Pull request by SiravitPhokeed**
<table>
<tr>
<td></td>
<td></td>
</tr>
</table>

**Features**
- API setup
- Landing
- Categories

**Fixes**
- The `dark:` Tailwind CSS prefix is erroneously on `class` mode

### CODE REVIEW

**Comment by smartwhatt** [comment]
do we deploy this?

**Comment by SiravitPhokeed** [comment]
No reason not to, I guess.

**Comment by smartwhatt** [comment]
> No reason not to, I guess.

Can you set that up? I don't have the envs

**Review by smartwhatt — COMMENTED** [review]
LGTM code wise, will approve after QA

File: utils/helpers/fetchJimmy.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+// Imports
+import { FetchReturn, Query } from "@/utils/types/fetch";
+import { Session } from "@supabase/supabase-js";
+import qs from "qs";
+import { sift } from "radash";
+
+/**
+ * @private Use `useJimmy` or `createJimmy` instead.
+ */
+export default async function fetchJimmy<Data extends {} | unknown = unknown>(
```
Comment (smartwhatt) [inline utils/helpers/fetchJimmy.ts:10]: Cool naming

File: utils/helpers/createJimmy.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Imports
+import fetchJimmy from "@/utils/helpers/fetchJimmy";
+import { createPagesServerClient } from "@supabase/auth-helpers-nextjs";
+import { Session } from "@supabase/supabase-js";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function createJimmy(
```
Comment (smartwhatt) [inline utils/helpers/createJimmy.ts:7]: Cool naming

File: utils/helpers/useJimmy.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// Imports
+import fetchJimmy from "@/utils/helpers/fetchJimmy";
+import { useSession } from "@supabase/auth-helpers-react";
+
+export default function useJimmy() {
```
Comment (smartwhatt) [inline utils/helpers/useJimmy.ts:5]: Cool naming

**Review by smartwhatt — APPROVED** [review]
LGTM!

**Pull request merged**